docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,145 @@
+# Integration with an AWS Kubernetes cluster
+This guide will walk you through how to use Calico Networking with an existing AWS-Kubernetes cluster.
+
+## Requirements
+* A working Kubernetes Deployment on AWS with one master and at least one minion
+    - While any AWS deployment will do, we recommend the Kubernetes [`kube-up` guide](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/getting-started-guides/aws.md) for AWS. This guide was created with the `kube-up` script in mind.
+* SSH access to your master and minions
+    - Unless otherwise specified, the `kube-up` script will create a `kube_aws_rsa` private key in the `~/.ssh` folder which you can use to access your AWS Instances.
+    - SSH in with the following command `ssh -i </path/to/key> ubuntu@<PUBLIC_IP>`
+
+## Preparing your master services
+On your master, download our etcd manifest
```

Comment:
This could use a little extra explanation to clarify that this is the etcd used by Calico, not Kubernetes.